### PR TITLE
Create schema.md

### PR DESCRIPTION
**Pull request by dgmccart**
Description of the data schema

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Portal Data Schema
+
+What are datasets, runs, and annotations in the Data Portal?
+
+1. A dataset is a community contributed set of image files for tilt series, reconstructed tomograms, and if available, cellular and/or subcellular annotation files. Every dataset contains only one sample type prepared and imaged with the same conditions. The dataset title, such as `S. pombe cryo-FIB lamellae acquired with defocus-only`, summarizes these conditions. Samples can be a cell, tissue or organism; intact organelle; in-vitro mixture of macromolecules or their complex; or in-silico synthetic data, where the experimental conditions are kept constant. Downloading a dataset downloads all files, including all avilable tilt series, tomograms, and annotations.
```
Comment (junxini) [inline docs/schema.md]: > including all avilable tilt series

minor spelling: available

File: docs/schema.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+2. A run is one experiment, or replicate, associated with a dataset, where all runs in a dataset are of the sample type, sample preparation, and imaging configuration. Typically a run produces only one tilt series. However, for a single tilt series, there can be multiple tomograms, each corresponding to the specific algorithm used to process the raw data. When downloading a run, you may download only the tomograms or the tomograms and available annotations.
+3. An annotation is a point or segmentation indicating the location of a macromolecular complex in the tomogram. All tomograms in the Data Portal have point annotations of (at least one) macromolecular complex, such an a ribosome. On the run page, you may choose to download tomograms with their annotations.
```
Comment (junxini) [inline docs/schema.md]: > such an a ribosome

minor spelling: such as a ribosome

I don't think all tomograms in the data portal will have point annotations. i think there will be some without any annotations. would be good to double confirm with Anchi

Comment (uermel) [reply docs/schema.md]: It's true that there will be datasets that won't have point annotations supplied by the data depositors. E.g. dataset 10010 will only have semantic segmentation masks for now.

Comment (dgmccart) [reply docs/schema.md]: Thank you for clarifying @uermel! I made a PR to fix that statement on the landing page [here.](https://github.com/chanzuckerberg/cryoet-data-portal/pull/164)

**Review by anchi2c — COMMENTED** [review]
Good job describing dataset.  For run,  The part starting with "However, for a single tilt series.", it should be changed.  When we have multiple tilt series in a run, i.e. mosaic case, one tomogram is generated from all of them with one processing algorithm.

**Comment by dgmccart** [comment]
Thanks @anchi2c I updated it to remove that sentence.

File: docs/cryoet_data_portal_docsite_schema.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+2. A run is one experiment, or replicate, associated with a dataset, where all runs in a dataset have the same sample and imaging conditions. Every run contains a collection of all tomography data and annotations related to imaging one physical location in a sample. It typically contains one tilt series and all associated data (e.g. movie frames, tilt series image stack, tomograms, annotations, and metadata), but in some cases, it may be a set of tilt series that form a mosaic. When downloading a run, you may download only the tomograms or the tomograms and available annotations.
+3. An annotation is a point or segmentation indicating the location of a macromolecular complex in the tomogram. On the run page, you may choose to download tomograms with their annotations.
+
+Descriptions of all terminology and metadata used in the portal is provided [here](https://docs.google.com/document/d/11h0u3YYF1EWCTjxu3ObShx26HgLAfJhn9I_tIaeQ6GI/edit#?usp=sharing).
```
Comment (richaagarwal) [inline docs/cryoet_data_portal_docsite_schema.md:9]: Will this be the final location or do we have plans to move this somewhere else?

**Review by richaagarwal — APPROVED** [review]

**Comment by dgmccart** [comment]
This content was incorporated into the FAQ page, so this PR is outdated.

**Pull request closed without merging**